examples/gnrc_border_router/uhcpc.c

```diff
@@ -0,0 +1,67 @@
+#include "uhcp.h"
```

Comment:
@kaspar030 is it not a cleaner approach to introduce `uhcpc` as a RIOT module? If somebody wants to implement another border router application, then including a module in the `Makefile` might look better, than creating a new symlink..